CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Activated Realm's annotation processor on connectedTest when the project is using kapt (#4008).
 * Fixed bug, preventing Sync client to renew the access token (#4038) (#4039).
 * Fixed "too many open files" issue (#4002).
+* Temporarily solution for #3561.
```

Comment:
Fair enough, what about `Added temporary work-around for bug crashing Samsung Tab 3 devices on startup`? The current description is meaningless if you glance over the change log.